packages/inter-protocol/test/psm/gov-replace-committee.js

```diff
@@ -78,44 +88,98 @@ const reserveThenDeposit = async (
   console.info('confirmed deposit for', debugName);
 };
 
-const invitePSMCommitteeMembers = async (
-  {
-    consume: {
-      namesByAddressAdmin,
-      economicCommitteeCreatorFacet,
-      econCharterKit,
-    },
-  },
-  { options: { voterAddresses = {} } },
+const handlehighPrioritySendersList = async ({
+  consume: { highPrioritySendersManager: highPrioritySendersManagerP },
+}) => {
+  const EC_HIGH_PRIORITY_SENDERS_NAMESPACE = 'economicCommittee';
+
+  const highPrioritySendersManager = await highPrioritySendersManagerP;
+
+  const voterAddressesToAdd = Object.values(
+    runConfig.economicCommitteeAddresses,
+  );
+  const voterAddressesToRemove = Object.values(
+    runConfig.economicCommitteeAddressesToRemove,
+  );
+
+  // Create Sets from the arrays for efficient lookup
+  const addSet = new Set(voterAddressesToAdd);
+  const removeSet = new Set(voterAddressesToRemove);
+
+  // Filter out common elements from both sets
+  const uniqueAddAddresses = voterAddressesToAdd.filter(
+    address => !removeSet.has(address),
+  );
+  const uniqueRemoveAddresses = voterAddressesToRemove.filter(
+    address => !addSet.has(address),
+  );
+
+  if (highPrioritySendersManager) {
+    // Add the addresses
+    for (let address of uniqueAddAddresses) {
+      await E(highPrioritySendersManager).add(
+        EC_HIGH_PRIORITY_SENDERS_NAMESPACE,
+        address,
+      );
+    }
+    // Remove the addresses
+    for (let address of uniqueRemoveAddresses) {
+      await E(highPrioritySendersManager).remove(
+        EC_HIGH_PRIORITY_SENDERS_NAMESPACE,
+        address,
+      );
+    }
+  }
+};
+
+const inviteECMembers = async (
+  { consume: { namesByAddressAdmin, economicCommitteeCreatorFacet } },
+  { options: { voterAddresses = {}, voterAddressesToRemove = {} } },
 ) => {
+  // Generate Invitations
```

Comment:
The invitations are generated only once, when the contract starts, so this comment should say "get generated invitations" or something similar.